### PR TITLE
feat(frontend): implement tip screen design

### PR DESCRIPTION
**Pull request by kateberryd**

### CODE REVIEW

File: JoyboyCommunity/src/modules/Tip/styles.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import {Spacing, ThemedStyleSheet} from '../../styles';
+
+export default ThemedStyleSheet((theme) => ({
+  modal: {
+    backgroundColor: theme.colors.surface,
+    paddingLeft: Spacing.medium,
+    paddingRight: Spacing.medium,
+    paddingBottom: Spacing.medium,
+    height: 728,
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/Tip/styles.ts:11]: Height should not be constant

File: JoyboyCommunity/src/modules/Tip/styles.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  icon: {
+    color: theme.colors.primary,
+  },
+
+  postHeader: {flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center'},
+
+  innerPostHeader: {flexDirection: 'row', gap: 5, alignItems: 'center'},
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/Tip/styles.ts:30]: Styles should not be single lines

File: JoyboyCommunity/src/modules/Tip/styles.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  title: {
+    marginBottom: Spacing.xsmall,
+  },
+  likes: {flexDirection: 'row', gap: 3, alignItems: 'center'},
+
+  recipient: {flexDirection: 'row', gap: 4, alignItems: 'center', paddingBottom: 10},
+  post: {
+    width: '100%',
+    height: 112,
+    backgroundColor: '#EC796B1A',
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/Tip/styles.ts:40]: This color already defined in the theme if im not wrong, it should be primaryLight

**Review by ugur-eren — CHANGES_REQUESTED** [review]
Thanks for the work! I've requested some changes. There are a lot requests but most of them are the same request

File: JoyboyCommunity/src/modules/Tip/index.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+      modalStyle={styles.modal}
+    >
+      <SafeAreaView edges={['bottom', 'left', 'right']}>
+        <View style={styles.card}>
+          <View style={styles.cardHeader}>
+            <View style={styles.cardContent}>
+              <Image
+                style={{width: 48, height: 48}}
+                source={require('../../assets/joyboy-logo.png')}
+              />
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/Tip/index.tsx:41]: You can use Avatar component, it provides polygon effect, and you can pass the size as `<Avatar size={48} />` directly.

File: JoyboyCommunity/src/modules/Tip/index.tsx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+            </View>
+          </View>
+
+          <Text fontSize={13} weight="medium" color="#14142C" style={{paddingTop: Spacing.small}}>
+            Live at Miami Stark Conf 2.0...
+          </Text>
+        </View>
+
+        <View style={{paddingTop: 40, paddingBottom: 40}}>
+          <View style={{marginBottom: 20}}>
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/Tip/index.tsx:64]: There shouldn't be inline styles, these should be moved to the styles.ts file

**Review by ugur-eren — CHANGES_REQUESTED** [review]
Thanks for the work! Only small details remaining, like moving inline styles to styles.ts file and using theme colors from useTheme hook. After that we can merge it.

**Review by ugur-eren — APPROVED** [review]
Thanks for the work! looks great

**Comment by kateberryd** [comment]
Thank you @ugur-eren

**Pull request merged**